pkg/container/qemu_runner.go

```diff
@@ -669,7 +669,7 @@ func createMicroVM(ctx context.Context, cfg *Config) error {
 		nil,
 		nil,
 		false,
-		[]string{"sh", "-c", "cp -r /mnt/. /home/build"},
+		[]string{"sh", "-c", "cp -a /mnt/. /home/build"},
```

Comment:
@89luca89 -- any reason we'd want `-r` instead of `-a` here?

We documented the reasoning when copying from the workspace -> locally but not here.

Reply:
Not really I think we can use `-a` as long as we keep reflinks, symlinks and hidden files :+1: